packages/config/config/devices/0x0346/keypad.json

```diff
@@ -0,0 +1,387 @@
+// Ring Keypad
+// Param information taken from https://support.ring.com/hc/en-us/article_attachments/360051851012/Keypad_Zwave_UK.pdf
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "Keypad",
+	"description": "Keypad",
```

Comment:
Can you describe this a little differently? Or omit the description. Is there a model number that should go for label instead?

Reply:
as far as I can see it is called `Alarm Keypad` => https://www.zwaveoutlet.com/products/ring-alarm-keypad

Reply:
Ring them self seems to call them both. In https://support.ring.com/hc/en-us/article_attachments/360051851012/Keypad_Zwave_UK.pdf they just call it Keypad. But then on https://support.ring.com/hc/en-us/articles/360040141092 which links to that PDF they call it Alarm Keypad. And I agree Alarm Keypad is more descriptive so I changed to that.